README.md

```diff
@@ -9,7 +9,7 @@ The following steps will install [Tekton Pipeline](https://github.com/tektoncd/p
     
 2. Apply Operator CRD
 
-    `kubectl apply -f deploy/crds/*_crd.yaml`
+    `kubectl apply -f deploy/crds/`
```

Comment:
If we turn to `ko`, the `crd` should also handle by `ko apply`, so the `crd` definition file should put in `/deploy/`

Reply:
We do not need to use ko on top of the crds, because there is no image building or pulling involved so far.

Reply:
@vincent-pli @houshengbo
I agree with vincent-pli
as we are providing `ko` support, it will be better if we can move all manifests to a single location and number them.

my only concern was the `operator-sdk olm-catalog gen-csv` command which generates a ClusterServiceVersion (CSV) file to be published on OperatorHub. By default, the command assumes the operator.yaml to be in deploy/ and crds in deploy/crd/.

However, the command can accept a config file where we can specify the paths,
eg: https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/olm-catalog/testdata/deploy/olm-catalog/csv-config.yaml

Reply:
@houshengbo could you make this update

in fact, if we decide to move to knative-pkg injection-controller, this part can stay the same.

Reply:
>  there is no image building or pulling involved so far.

true, but if we keep them separate we will have to do a `kubectl apply -f <crd path>` and `ko apply config`
i would prefer haveing a single directory with numbered files

Reply:
@nikhil-thomas I can use the file link instead of the physical file. We can still keep crd under deploy/crds, but file-link it under config. openrator-sdk works with deploy dir and ko works with config dir.